src/applications/personalization/view-dependents/components/ViewDependentsHeader/ViewDependentsHeader.jsx

```diff
@@ -3,14 +3,14 @@ import React from 'react';
 const ViewDependentsHeader = () => (
   <div className="vads-l-row">
     <div className="vads-l-col--12">
-      <h1>Your Dependents</h1>
+      <h1>Your VA Dependents</h1>
       <p className="vads-u-font-size--md vads-u-font-family--serif">
-        Below is the information we have for your dependents. You can file a
-        claim for more disability compensation whenever you add a dependent. If
-        something changes with a dependent's status, you should notify VA.
+        Below is a list of dependents we have on file for you. You can file a
+        claim for additional disability compensation whenever you add a new
+        dependent.
       </p>
       <a
-        href="/disability-benefits/apply/dependents"
+        href="https://www.ebenefits.va.gov/ebenefits/about/feature?feature=dependent-compensation"
```

Comment:
An `aria-label` and a `title` attribute could be added to this link. Since it's external it may also need `target="_blank"` and `rel="noopener noreferrer"`

Reply:
I'm not sure that an `aria-label` would do much good here.  From a [doc from Mozilla](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/WAI-ARIA_basics):

> Ideally, you should always use native HTML features to provide the semantics required by screenreaders to tell their users what is going on. 

The example on [this wiki page](https://www.w3.org/WAI/GL/wiki/Using_aria-label_for_link_purpose) adds an `aria-label` when the link text isn't descriptive, like "Read more".  In the situation here, "Add or remove a dependent" seems descriptive enough, but I say all this with the caveat that I'm not an accessibility expert.

Reply:
I added both the label and title tags. I didn't want to add a I asked Jim about this link opening in a new tab and he said it should not.

Reply:
I added both the aria-label and the title but I asked and Jim said the link should not open in a new tab so we are good there